### ISSUE TITLE
2nd carousel doesn't work on the same page.

### ISSUE THREAD

**Issue by Hayko1994** [OwlCarousel2/OwlCarousel2#1938]
I use 2 carousels on the same page first one for screenshots section 2nd one for testimonials section. On the first one autoPlay is 1000 while autoPlay doesn't work. The 2nd carousel doesn't work fully. Its items are located near the each other and there is not any indicator or animation. 2md one is like just columns and nothing else. Can you help me to solve that trouble?

**Comment by keefyhub** [comment]
Can you supply the code? Also are you getting any errors in the console?

**Comment by Hayko1994** [comment]
Here is the link on Codepen. I have not included images but it still visible. 
[https://codepen.io/Hayko/pen/yXapXE](url)
autoplay doesn't work fro both while the second one doesn't work fully.

**Comment by keefyhub** [comment]
it looks like you have some invalid options - https://owlcarousel2.github.io/OwlCarousel2/docs/api-options.html

`autoPlay` should be `autoplay` and your responsive stuff should be something like this

```
  responsive: {
      0: {
        items: 1
      },
      1025: {
        items: 2
      }
    },
```

On the 2nd carousel, you are missing the `owl-theme` class so there is no styles for the dots.

Im not really sure why, but it doesnt look like the bootstrap offset columns class is working before your carousel either, so the carousel appears to be off to the left (not full width)

**Comment by Hayko1994** [comment]
I haven't got the last sentence. Now 2 carousels work but when you go to the other blank of the browser(page) and then come back loop doesn't work anymore until the doing refresh.

**Comment by keefyhub** [comment]
You are using these classes `col-sm-offset-1 col-sm-10` and the offset is not working, so the carousel appears to be off to the left.

For the autoplay not working, do you mean if you navigate away from the page, then press the back button, the autoplay doesnt work? Im not seeing that on the codepen example.

**Comment by Hayko1994** [comment]
Nope. I mean when there are several tabs are opened in the browser. You click on the other tab and then after some period again click the tab where that page was run. In this case code pen.  Like there are 2 tabs. on the 2nd one was opened google e.g.  Click on that tab and after some period click to the tab where code pen is opened and loop will not work anymore.

**Comment by keefyhub** [comment]
ah I see, looks like this has already been reported - https://github.com/OwlCarousel2/OwlCarousel2/issues/1727

There are work arounds in that thread, but nothing has been merged yet

**Comment by Hayko1994** [comment]
Hope it will be fixed soon :)
